ql/src/java/org/apache/hadoop/hive/ql/exec/vector/expressions/StringGroupConcatColCol.java

```diff
@@ -320,7 +320,8 @@ public void evaluate(VectorizedRowBatch batch) throws HiveException {
       /*
        * Do careful maintenance of the outputColVector.noNulls flag.
        */
-
+      // propagate not nulls flag.
+      outV.noNulls = true;
```

Comment:
@maheshrajus this assignment was there earlier and got removed as a part of [HIVE-18622](https://issues.apache.org/jira/browse/HIVE-18622). Now you are adding it again so please check carefully why it was removed as a part of [HIVE-18622](https://issues.apache.org/jira/browse/HIVE-18622).

Reply:
@mdayakar thanks for review. I will check reason behind it.